aws/websites/metrics.pytorch.org/files/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.7'
+
+services:
+  grafana:
+    image: grafana/grafana:8.0.6
+    logging:
+      driver: "json-file"
```

Comment:
since it's already in aws, cloudwatch driver for the logging may work better, https://docs.docker.com/config/containers/logging/awslogs/

the grafana itself may also need some monitoring

Reply:
Added it to #69 to leave for a follow-up